packages/core/README.md

```diff
@@ -18,23 +17,15 @@ This repository contains the static HTML, Javascript, CSS, and images content of
 
 -   [Modules docs](docs/Modules/README.md) - Contains implementation guides (i.e., scaffolding help, etc.)
 
-## Use a custom domain
-
-To use your custom domain, please put it in a file named `CNAME` inside the `scripts` folder of your local clone of deriv-app.
-
 ## How to work with this project
 
 ### Deploy to your gh-pages for the first time
 
-1.  Register your application [here](https://developers.binary.com/applications/). This will give you the ability to redirect back to your Github pages after login. Use `https://YOUR_GITHUB_USERNAME.github.io/deriv-app/` for the Redirect URL and `https://YOUR_GITHUB_USERNAME.github.io/deriv-app/en/redirect` for the Verification URL. If you're using a custom domain, replace the Github URLs above with your domain and remove the `deriv-app` base path.
-
-2.  In `src/config.js`: Insert the `Application ID` of your registered application in `user_app_id`.
```

Comment:
I think this also can be deleted because If I understood everything correctly core package generates automatically AP ID, also this path `src/config.js` is not correct and there is nothing here.

Reply:
this one is for redirection from deriv.com login right into developer's custom test link